pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1773,7 +1782,7 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
 
     await this.progressTracker.action('Starting virtual machine', 100, async() => {
       try {
-        await this.lima('start', '--tty=false', await this.isRegistered ? MACHINE_NAME : this.CONFIG_PATH);
+        await this.lima({ LIMA_SSH_PORT_FORWARDER: 'true' }, 'start', '--tty=false', await this.isRegistered ? MACHINE_NAME : this.CONFIG_PATH);
```

Comment:
This is setting the variable unconditionally instead of checking `this.cfg.experimental.sshPortForwarder`. But if you make the other suggested change, then nothing needs to be done here.

Reply:
:headdesk: Thanks for the catch. I don't know why when I tested with it off it didn't show up…